oauth2-server/src/main/java/com/clouway/oauth2/client/CertificatesNotConfiguredException.java

```diff
@@ -0,0 +1,8 @@
+package com.clouway.oauth2.client;
+
+/**
+ * @author Vasil Mitov <v.mitov.clouway@gmail.com>
+ */
+public class CertificatesNotConfiguredException extends RuntimeException {
```

Comment:
Leftover exception that was a part of my ideas for solving the chaos that were the tests. Removed.